docs/export.md

```diff
@@ -0,0 +1,38 @@
+# Export service
+Export service is build to bridge between two Mainflux instances. For example we can run Mainflux on the gateway and
+we want to export the data to cloud instance of Mainflux as well.
+Export service is listening to the NATS and forwards payload to the specified MQTT channel.
+ 
+
+## Run Export service
+
+Easiest way to run export service is 
+
+```
+git clone github.com/mainflux/export
+cd github.com/mainflux/export
+make
+cd build
+./mainflux-export
+```
+
+service will pickup config file  `github.com/mainflux/export/configs/config.toml`
```

Comment:
Are you sure it will pick it up if you move to `build`, or you should do `./build/mainflux-export`?